pre-docs/inventory_version2.md

```diff
@@ -180,6 +180,77 @@ groups:
             hostname: facts.networking.interfaces.en0.ipaddress
 ```
 
+#### Task
+
+The Task plugin will run a task on `localhost` to lookup configuration
+information or target lists for inventory..
+
+For both use cases the plugin accepts two keys:
+* `task`: The task to run.
+* `parameters`: The parameters to pass to the task.
+
+example:
```

Comment:
```suggestion
For example, this would equivalent to running `bolt task run my_json_file::targets -n localhost file=/etc/targets/data.json environment=production app=my_app`:
```